EventGenerator/src/SConscript

```diff
@@ -13,6 +13,8 @@ helper=mu2e_helper(env);
 
 cryinc = os.environ['CRYHOME'] + '/src'
 crylib = os.environ['CRY_LIB']
+g4inc = os.environ['G4INCLUDE']
```

Comment:
I would like to keep all references to G4 within Mu2eG4.  The motivation is to be able to make G4-free builds which are required by the trigger.  We eventually want to get to MC-free builds but we are not there yet. 

  One generator module is already in G4 for this reason:

  Mu2eG4/src/GammaConvGenerator_module.cc

Please look into what that would require for this PR.  

Maybe it's time to make a new top level directory for event generators that depend on G4.?

Reply:
The fundamental "problem" is that materials are defined via hard-codes in `Mu2e/src/ConstructMaterials.cc`, so that interfacing with G4 is the only way to query what they are comprised of. If there were a Mu2e-owned `Service`/interface which maintained a table of configurable materials and their compositions, which G4 queried when constructing `G4Material`s, then that would provide the functionality I'm looking for (and presumably obviate the delayed-initialization tricks I've had to use here).

But I imagine that there is a reason that things are done the way they are (and if not, implementing the kind of ting I'm imagining is a separate project). In that case, I don't particularly care whether the generator lives under `EventGenerator/`, `Mu2eG4/`, or somewhere new. Unless there is some subtly in crossing project boundaries that I'm aware of, just moving it into `Mu2eG4` would probably satisfy the separation you'd like.

Reply:
Please move it into Mu2eG4 and revert the changes to EventGenerator/src/SConscript.

Implementing a complete parallel materials system seemed like far too much work for something that would never be used except to feed G4.  And we did not have a use case to drive making a simplified materials system.  It's only been in the past year or so that we have had a use case.

Reply:
Ok --- `CompositeMaterialGenerator`, `ElementSamplerTool`, and `AtomicVolumeSamplerTool`, which depend directly or indirectly on G4, now live under `Mu2eG4`, and the uses of G4 libraries or `Mu2eG4` in `EventGenerator/src/SConscript` have been removed. `PositionSamplerTool` and `MuStopPositionSamplerTool`, which do not depend on G4, still live under `EventGenerator`.